test/katex-spec.js

```diff
@@ -715,7 +715,7 @@ describe("A text parser", function() {
     const textExpression = "\\text{a b}";
     const noBraceTextExpression = "\\text x";
     const nestedTextExpression =
-        "\\text{a {b} \\blue{c} \\textcolor{#fff}{x} \\llap{x}}";
+        "\\text{a {b} \\blue{c} \\textcolor{#fff}{x} \\mathllap{x}}";
```

Comment:
Why change all `\lap`s in this file to `\mathlap`s?

Reply:
It seemed to me that `\mathllap` is a core command. I found `\llap` to be less interesting. But your point is well taken. This line is supposed to be testing a *text* parser.  I'll change it back.

Reply:
Well, now I have a better answer to your question about the test. Since `\lap` is a test mode function, we should not expect it to work properly when given math mode arguments, such as `\left`. So the tests really should be revised to test `\mathllap` and not `\lllap`.